src/main/java/org/example/controllers/EmployeeController.java

```diff
@@ -0,0 +1,40 @@
+package org.example.controllers;
+
+import io.swagger.annotations.Api;
+import org.example.exceptions.FailedToCreateException;
+import org.example.exceptions.InvalidException;
+import org.example.models.EmployeeRequest;
+import org.example.services.EmployeeService;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.sql.SQLException;
+
+@Api("Engineering Academy BatchMasters")
+@Path("api/employees")
+public class EmployeeController {
+
+    EmployeeService employeeService;
+
+    public EmployeeController(final EmployeeService employeeService) {
+        this.employeeService = employeeService;
+    }
+
+    @POST
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response createEmployee(final EmployeeRequest employeeRequest) {
```

Comment:
requirement 1 asks for the ability to create a delivery employee but this appears to be a generic employee?